app/dashboard/management/commands/pull_ipfs_hashes.py

```diff
@@ -0,0 +1,119 @@
+# -*- coding: utf-8 -*-
+"""Handle IPFS hash pulling command logic.
+
+Copyright (C) 2018 Gitcoin Core
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as published
+by the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""
+import logging
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+from dashboard.helpers import UnsupportedSchemaException
+from dashboard.utils import BountyNotFoundException, get_ipfs, getBountyContract, infura_ipfs_pin, ipfs_pin
+
+logger = logging.getLogger(__name__)
+default_start_id = 0 if not settings.DEBUG else 402
+
+
+def get_bounty_id(_id, network):
```

Comment:
I am assuming you mean because it's from sync_geth?  We definitely can.  I was trying to avoid diving down that rabbit hole until after we resolve the pinning situation afflicting BN, but we should probably refactor sync_geth while we're at it.

Reply:
yes